<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ArxDialog</class>
 <widget class="QDialog" name="ArxDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>291</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>271</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>5</y>
      <width>261</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMin"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelB">
       <property name="text">
        <string>B</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QCheckBox" name="checkBoxOgraniczenie">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="labelOgraniczenie">
       <property name="text">
        <string>Ograniczenia</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditA"/>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxOpoznienie"/>
     </item>
     <item row="5" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxMax"/>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxSzum"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSzum">
       <property name="text">
        <string>Szum</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelMin">
       <property name="text">
        <string>Min</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditB"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelMax">
       <property name="text">
        <string>Max</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelA">
       <property name="text">
        <string>A</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelOpoznienie">
       <property name="text">
        <string>Opóźnienie</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Anuluj</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
